### PR TITLE
Improves README, removes deprecated/duplicated information

### PR DESCRIPTION
**Pull request by pabrus**
- Removes ASC folder as everything is deployed by the single ARM template now
- Improves root README and adds information from deleted folders/READMEs
- References specific README files from the root README

### CODE REVIEW

File: aad-onboarding/azuread-emergency-access/usage-alert.md [hunk 1]
```diff
@@ -2,23 +2,22 @@
 
 1. [Send Azure AD sign-in logs to Azure Monitor.](https://docs.microsoft.com/azure/active-directory/reports-monitoring/howto-integrate-activity-logs-with-log-analytics)
 
-1. Object Object IDs for each break-glass account
+2. Object Object IDs for each break-glass account
 
-1. Create the following log search alert.
+3. Create the following log search alert.
```
Comment (ckittel) [inline aad-onboarding/azuread-emergency-access/usage-alert.md:7]: just as an FYI, ordered lists are best kept as 1. 1. 1. ... to support easy insertion and rearranging of entries, at render time they show up as an <ol> and the number takes care of itself.  Nothing to change, just wanted to let you know that you don't need to make changes like this, generally speaking.

File: README.md [hunk 2]
```diff
@@ -1,49 +1,147 @@
...
+## How to evolve later
+
+If the business requirements change over time and/or your organization grows, this simple implementation won't be enough to guarantee best practices and a more complex solution is recommended.
+
+Please refer to [Enterprise-Scale Landing Zones](https://github.com/Azure/Enterprise-Scale) for different alternatives on how to evolve.
+
+## Pre-requisites
+
+- To deploy this ARM template, your user/service principal must have Owner permission at the selected subscription.
+- It is **highly** recommended to use a new Subscription. This template will change security configuration and policies that will affect any pre-existent configuration or resource.
```
Comment (ckittel) [inline README.md:24]:
```suggestion
- It is **highly** recommended to use a new Subscription, not yet being used for production resources. This template will change security configuration and policies that might affect any pre-existing configurations or resources.
```

File: README.md [hunk 3]
```diff
@@ -1,49 +1,147 @@
...
 
 CAF consists of multiple methodologies, providing the right guidance at the right time in an organization's journey. The **Secure** methodology provides security guidance by providing clarity for the processes, best practices, models, and experiences. This guidance is based on the lessons learned and real world experiences of real customers, Microsoft's security journey, and work with organizations like NIST, The Open Group, and the Center for Internet Security (CIS).
 
-## Secure Accelerator
+## Customer profile
+
+This workshop provides a design path and initial technical state for Small Enterprises or Customers willing to know and start using Azure Security Tools and Services. It is meant for organizations that do not have a large IT team and do not require fine grained administration delegation models. Hence, all resources and security rules are consolidated in a single subscription.
```
Comment (ckittel) [inline README.md:11]:
```suggestion
This workshop provides a design path and initial technical state for small enterprises or customers willing to learn and start using Azure security tools and services. It's meant for organizations that do not yet have a large IT team nor require fine grained administration delegation models. Hence, all resources and security rules are consolidated in a single subscription.
```

File: README.md [hunk 4]
```diff
@@ -1,49 +1,147 @@
...
+
+This reference implementation is also well suited for bigger organization or customers who want to start with Azure Security Center for their net new deployment/development in Azure by implementing a network architecture based on the traditional hub and spoke network topology.
```
Comment (ckittel) [inline README.md:13]:
```suggestion
This reference implementation is also well suited for bigger organization or customers who want to start with Azure Security Center for their net new deployment/development in Azure by implementing a network architecture based on the traditional hub-spoke network topology.
```

**Review by ckittel — APPROVED** [review]
Just a couple minor edits.  Apply what you wish, then merge when ready.  Thanks.

File: aad-onboarding/azuread-emergency-access/usage-alert.md (hunk 1 above)
Comment (pabrus) [reply aad-onboarding/azuread-emergency-access/usage-alert.md:7]: Ahh... I didn't know. I will put back original numbering in the next PR.

**Pull request merged**